src/page-templates/main-content-page/content-page-template.tsx

```diff
@@ -314,6 +361,11 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
                     )}
                 </div>
                 {!previewMode && ratingSection}
+                {(githubUrl || liveSiteUrl) &&
```

Comment:
Might be a good idea to throw in an `isCodeExample` condition on this and line 401, just to be super safe in case a `githubUrl` or `liveSiteUrl` accidentally gets added to a non-Code Example piece.